src/signals/incident-management/containers/IncidentDetail/components/LocationForm/index.js

```diff
@@ -108,22 +118,18 @@ class LocationForm extends React.Component { // eslint-disable-line react/prefer
 
 LocationForm.defaultProps = {
   location: {},
-  newLocation: {},
-  incidentModel: {
-    incident: {},
-    patching: {
-      location: false
-    }
-  }
+  newLocation: {}
 };
 
 LocationForm.propTypes = {
-  id: PropTypes.string,
-  incidentModel: PropTypes.object,
+  incident: PropTypes.object.isRequired,
```

Comment:
Should be `incident: PropTypes.shape({}).isRequired,` where at least `id` and `location` are defined as inner prop types.

Reply:
ze zijn allemaal met correcte prototypes nu